tests/unit/utils/test_populate.py

```diff
@@ -31,42 +31,159 @@ def test_create_settings_from_fixture(database):
 
 
 def test_update_settings_from_fixture(database):
```

Comment:
Let's bust this test into individual ones. Other than that, everything else looks good.

Reply:
I'm not familiar with test.py, but my understanding was that individual test functions are independent. Or can they call each other and then only the top-most one is executed automatically?

Reply:
Individual tests should be independent. I'm asking to take this huge test and break it into multiple, simpler tests.